libtenzir/include/tenzir/actors.hpp

```diff
@@ -234,7 +234,9 @@ using importer_actor = typed_actor_fwd<
   auto(stream_sink_actor<table_slice>)
     ->caf::result<caf::outbound_stream_slot<table_slice>>,
   // Register a FLUSH LISTENER actor.
-  auto(atom::subscribe, atom::flush, flush_listener_actor)->caf::result<void>>
+  auto(atom::subscribe, atom::flush, flush_listener_actor)->caf::result<void>,
+  // Decomissions all active partitions, effectively flushing them to disk.
```

Comment:
The importer doesn't know about partitions, so we should not talk about them here.
```suggestion
  // Push buffered slices downstream to make the data available.
```